### ISSUE TITLE
Using comment in JSX expression produce parsing error

### ISSUE THREAD

**Issue by chatellier** [jruchaud/gulp-gettext#1]
Hi,

Using a JSX comment synthax produce a strange code were parsing is failing.

JSX code look like:

```
return (
    <div>
        {/* whatever */}
    </div>
);
```

at runtime, this error is throw:

```
/home/chatellier/projets/xxx/gulp/ServerTask.js:45
            throw e;
                  ^
TypeError: undefined is not a function
    at c (/home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/node_modules/acorn/dist/walk.js:38:15)
    at Object.JSXExpressionContainer (/home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/index.js:37:9)
    at c (/home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/node_modules/acorn/dist/walk.js:38:15)
    at /home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/index.js:33:13
    at Array.forEach (native)
    at Object.JSXElement (/home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/index.js:32:23)
    at c (/home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/node_modules/acorn/dist/walk.js:38:15)
    at /home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/index.js:33:13
    at Array.forEach (native)
    at Object.JSXElement (/home/chatellier/projets/xxx/node_modules/gulp-gettext-parser/index.js:32:23)
```

**Comment by jruchaud** [comment]
Hi
I think the issue come from the JSX parser. I will checked.

**Comment by faxioman** [comment]
Hi.
Yes, it's a bug of acorn-jsx, fixed as reported here: https://github.com/RReverser/acorn-jsx/pull/31.
An update to the package.json dependency would be great! ;)

**Comment by jruchaud** [comment]
Hi
Thanks. I have updated the acorn-jsx and fixed an issue in walker for acorn. Now it works and available in version 1.8.
